utils/isPhoneUser.ts

```diff
@@ -0,0 +1,15 @@
+// CHECK THIS, onko tässä mitään järkeä?
+
+// Ei taida olla tarpeellinen, jos tämä tarkistus tehdään ainoastaan _app.tsx-tiedostossa. Turhaan on oma tiedostonsa tälle
+
+export function isPhoneUser(window?: Window & typeof globalThis): boolean {
+  if (window) {
+    const screenWidth = window.screen.width;
+    const screenHeight = window.screen.height;
+    if (screenWidth < 768 || screenHeight < 768) {
+      return true;
+    }
+    return false;
+  }
+  return false;
+}
```

Comment:
1. Onko tarpeellista olla omassa tiedostossaan? On käytössä vain `/pages/_app.tsx`-tiedostossa
2. Parempi tapa selvittää onko käyttäjä tietokoneella tai puhelimella lienee `userAgent`ista tiedon saaminen vs tämä nykyinen tapa.

3. Jos tämä tapa on tässäkohtaa ok, niin tekisin seuraavan pienen muutoksen koodiin:

```suggestion
// CHECK THIS, onko tässä mitään järkeä?

// Ei taida olla tarpeellinen, jos tämä tarkistus tehdään ainoastaan _app.tsx-tiedostossa. Turhaan on oma tiedostonsa tälle

export function isPhoneUser(window?: Window & typeof globalThis): boolean {
  if (window) {
    if (window.screen.width < 768 || window.screen.height < 768) {
      return true;
    }
    return false;
  }
  return false;
}
```